CONTRIBUTING.md

```diff
@@ -2,6 +2,43 @@
 
 You'd like to help? Great!  :tada:
 
+All contributions to `PyPRECIS` are made via merges with the develop branch.
+New contributors should add their details to the "code contributors" section of this document as part of their first request.
+The developer who reviews each pull request is responsible for checking that the contributor's name is listed in this file before merging the changes into master branch.
+
+## Code Contributors  
+
+ *  Hamish Steptoe (Met Office, UK), [hamish.steptoe@metoffice.gov.uk](mailto:hamish.steptoe@metoffice.gov.uk)
```

Comment:
I'd suggest we can link to github profiles (like with Iris [e.g.](https://scitools-iris.readthedocs.io/en/stable/whatsnew/index.html#iris-whatsnew)) and then remove emails so they can't be scraped for spam. Is there a particular reason we should share individual developer emails?  There is a generic PRECIS mailbox below and all other interaction should probably be via issues, pull requests or the [discussion board](https://github.com/MetOffice/PyPRECIS/discussions)
```suggestion
 *  @hsteptoe (Met Office, UK) 
```

Reply:
@cpelly  can you comment on this? I based this one off the example you linked